src/serialize.h

```diff
@@ -673,6 +723,16 @@ inline void Unserialize(Stream& is, T&& a)
     a.Unserialize(is);
 }
 
+/** Identity formatter. Serializes objects as themselves. */
```

Comment:
Maybe add a comment when this is to be used, on first glance it seems like a no-op.

Reply:
Done.